src/Workspaces/Core/Portable/SymbolSearch/SymbolSearchLogService.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.SymbolSearch
+{
+    internal sealed class SymbolSearchLogService : ISymbolSearchLogService
```

Comment:
this is just a move of the existing service.  but the cleanup and simplification makes it look new.